src/main/java/org/thoughtcrime/securesms/components/AvatarImageView.java

```diff
@@ -30,11 +29,6 @@ public AvatarImageView(Context context) {
   public AvatarImageView(Context context, AttributeSet attrs) {
     super(context, attrs);
     setScaleType(ScaleType.CENTER_CROP);
-
-    if (attrs != null) {
-      TypedArray typedArray = context.getTheme().obtainStyledAttributes(attrs, R.styleable.AvatarImageView, 0, 0);
-      typedArray.recycle();
-    }
```

Comment:
pointless statement, loading some attributes but not actually using them, probably leftovers after the code actually using it got removed